alvr/server_core/src/connection.rs

```diff
@@ -540,13 +540,16 @@ fn connection_pipeline(
         Err(e) => return Err(e),
     };
 
+    let headset_is_worn_initial_state;
```

Comment:
This variable doesn't need to exist, you can just use `streaming_caps.headset_is_worn`